swagger.yaml

```diff
@@ -1581,7 +1584,7 @@ components:
           type: string
           description: Estimated duration of the task
         estimatedReadiness:
-          type: string
+          type: number
           description: Estimated readiness of the task
```

Comment:
Add a comment with its range?
I think swagger ha the keywords for it https://swagger.io/docs/specification/data-models/data-types/

Reply:
Range added